src/topic_training/src/Begin_Here.md

```diff
@@ -0,0 +1,122 @@
+# WRoverPlayground - Topic Training
+Now that you have completed the Tools Training, you are ready to learn about ROS topics. From the Setup Training, you should be familiar with:
+- the basics of ROS Nodes
+- rosnode
+- and roscore.
+
+Once you complete this training, you should be familiar with:
+- the purpose of topics
+- the relationship between Publishers and Subscribers
+- setting up Publishers and Subscribers
+- running Publishers and Subscribers
+- rqt_graph
+- rqt_plot
+- custom topic types
+- and rostopic type {topic}.
+
+## The Purpose of Topics
+The word "topic" sounds very general, but it is actually a specific concept that allows ROS nodes to communicate with each other. This is extremely important because one node cannot possibly contain all the information a rover would need.
```

Comment:
It could, is the thing.  This is typically called a 'monolith', and becomes very hard to maintain without extreme diligence.  They are often much more complex as well.

I would argue that the topic is a way to stream data out of or into a program.

Reply:
Fixed in f580517.

Reply:
I think this a very good improvement and is a good view of topics.  Nice job!